### PR TITLE
adding subdomain for MornCast

### PR DESCRIPTION
**Pull request by brunoblaise**

### CODE REVIEW

**Comment by brunoblaise** [comment]
for Morncast club.

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2069,3 +2069,12 @@ zwp:
   - ttl: 1
     type: A
     value: 172.99.233.117
+ morncast:
+  - ttl: 1
+    type: CNAME
+    value: hackathon-coral.vercel.app.
+  _vercel:
```
Comment (sampoder) [inline hackclub.com.yaml:2076]: Hey! Please add these to Vercel record at the top.

Comment (brunoblaise) [reply hackclub.com.yaml]: fixed it.

File: hackclub.com.yaml [hunk 2]
```diff
@@ -2086,3 +2086,7 @@ zwp:
   - ttl: 1
     type: A
     value: 172.99.233.117
+ morncast:
+  - ttl: 1
+    type: CNAME
+    value: hackathon-coral.vercel.app.
```
Comment (sampoder) [inline hackclub.com.yaml:2092]: This should be Vercel's CNAME address

**Review by sampoder — CHANGES_REQUESTED** [review]

**Comment by exu3** [comment]
Closing because this PR has been inactive for a month. Feel free to re-open if still interested.

**Pull request closed without merging**